### ISSUE TITLE
Do not display missing imags in threads list

### ISSUE THREAD

**Issue by numeroteca** [numeroteca/pageonex#185]
In the threads list at /threads the missing images are being displayed as broken images (chrome) or the alt text (firefox). We should add a filter to avoid this behavior and prevent from showing these missing images.

I tried to add a validation in the view with `
<% if File.exist?(image.thumbnail_local_path) %>` or with 
`image.codeable` but they didn't work. I must be missing something. ideas?
